dotfiles_template/shell/zsh/.zshenv

```diff
@@ -1,4 +1,4 @@
 export DOTFILES_PATH="XXX_DOTFILES_PATH_XXX"
 export DOTLY_PATH="$DOTFILES_PATH/modules/dotly"
 export DOTLY_THEME="codely"
-export ZIM_HOME="$DOTLY_PATH/modules/zimfw"
+export ZIM_HOME="$DOTFILES_PATH/.zim"
```

Comment:
Not sure where the ZIM_HOME would make more sense. It's not a git submodule now, so I think the old path is not the best one anymore.